### PR TITLE
fix: put custom input length in form

### PR DESCRIPTION
**Pull request by MaiaS**
## WHAT IT DOES:

_Description of the pull request, what changed_
puts custom length input in form

### CODE REVIEW

File: src/components/Input/InputField/SongInputField.tsx [hunk 1]
```diff
@@ -24,22 +24,39 @@ const SongInputField = ({
...
+          <SummarizeButton>Summarize</SummarizeButton>
+        </div>
+        <SummaryLengthSlider
+          summaryLength={summaryLength}
+          setSummaryLength={setSummaryLength}
+          customLength={customLength}
+          setCustomLength={setCustomLength}
         />
         <input
           className="mb-12 h-20 rounded-full border-2 border-primary px-4 text-center text-base font-medium transition-all duration-200 placeholder:font-normal placeholder:!text-dark/70  focus:border-primary focus:outline-none focus:ring-2 focus:ring-inset md:h-[5.7rem]"
```
Comment (pratikmdhr) [inline src/components/Input/InputField/SongInputField.tsx]: this looks like an extra input field?

File: src/components/Input/InputField/SongInputField.tsx [hunk 2]
```diff
@@ -52,12 +69,6 @@ const SongInputField = ({
         />
         <SummarizeButton>Summarize</SummarizeButton>
```
Comment (pratikmdhr) [inline src/components/Input/InputField/SongInputField.tsx]: extra button?

**Review by pratikmdhr — CHANGES_REQUESTED** [review]
Hey looks like there are some extra input field and summarize button for the song, other than that looks good!

File: src/components/Input/InputField/SongInputField.tsx (hunk 2 above)
Comment (MaiaS) [reply src/components/Input/InputField/SongInputField.tsx]: Whoops! Nice catch my bad!

**Review by pratikmdhr — APPROVED** [review]
💯

**Pull request merged**